operator-framework/src/main/java/io/javaoperatorsdk/operator/processing/ExecutionConsumer.java

```diff
@@ -22,7 +22,7 @@
 
   @Override
   public void run() {
-    PostExecutionControl postExecutionControl = eventDispatcher.handleEvent(executionScope);
+    PostExecutionControl postExecutionControl = eventDispatcher.handleExecution(executionScope);
```

Comment:
just loud thinking, wouldn't the code be simpler if we had 2 ExecutionConsumers called 
SimpleConsumer and RetryableConsumer and let the consumer to take care of the retrying rather than the handler?

Reply:
@psycho-ir thing is this is closely relates with the events are comming in while the controller is executed, that layer does not have enough info basically now. (Also its just really a wraper around executor).